ng2-components/ng2-activiti-tasklist/src/components/activiti-people-search.component.ts

```diff
@@ -85,17 +84,38 @@ export class ActivitiPeopleSearch implements OnInit, AfterViewInit {
         return isUpgraded;
     }
 
-    onRowClick(userClicked: User) {
-        this.onRowClicked.emit(userClicked);
-        this.userList = this.userList.filter((user) => {
+    onRowClick(user: User) {
+        this.selectedUser = user;
+    }
+
+    closeSearchList() {
+        this.success.emit();
+    }
+
+    addInvolvedUser() {
+        if (this.selectedUser === undefined) {
+            return;
+        }
+        this.success.emit(this.selectedUser);
+        this.users = this.users.filter((user) => {
             this.searchUser.reset();
-            return user.id !== userClicked.id;
+            return user.id !== this.selectedUser.id;
         });
     }
 
     getDisplayUser(user: User): string {
         let firstName = user.firstName && user.firstName !== 'null' ? user.firstName : 'N/A';
         let lastName = user.lastName && user.lastName !== 'null' ? user.lastName : 'N/A';
-        return firstName + ' - ' + lastName;
+        return firstName + ' ' + lastName;
+    }
+
+    getShortName(user: User): string {
```

Comment:
I think we should improve this part. 
getDisplayUser and getShortName do basically the same operations, maybe merging them together and adding in input a delimitator could allow to simplify this part.